Library/Homebrew/cmd/info.rb

```diff
@@ -62,16 +62,18 @@ def github_fork
   end
 
   def github_info f
-    if f.tap?
-      user, repo = f.tap.split("/", 2)
```

Comment:
Why should downstream code need to know that the tap name is determined by regex matching the path to the formula's location on disk, when it can just ask the formula for the tap name?